managed-ledger/src/test/java/org/apache/bookkeeper/test/BookKeeperClusterTestCase.java

```diff
@@ -226,6 +226,8 @@ public void tearDown() throws Exception {
         if (tearDownException != null) {
             throw tearDownException;
         }
+        // cleanup for metrics.
+        metadataStore.close();
```

Comment:
this should be done before killing zookeeper, can you add it in stopZKCluster before `zkUtil.killCluster();` ?